ledger/src/shred.rs

```diff
@@ -64,6 +64,8 @@ pub use {
         stats::{ProcessShredsStats, ShredFetchStats},
     },
     crate::shredder::{ReedSolomonCache, Shredder},
+    solana_keypair::Keypair,
```

Comment:
not a bug - these are needed for expanded macro of set_signature and the function that uses it.

Reply:
good catch. I somehow missed line 59 and thought this upper use block was dcou